example.js

```diff
@@ -376,14 +376,28 @@ client.on('message', async msg => {
             requesterIds: ['number1@c.us', 'number2@c.us'],
             sleep: null
         });
+    } else {
+        // Pins a message
```

Comment:
I think it would be better to default to WhatsApp defaults . But allow the user to set to any amount, just disclose that the defaults are 24hours, 7 days, 30 days.

Reply:
Agree with you, I'll make some changes

Reply:
@tuyuribr 
[Done](https://github.com/pedroslopez/whatsapp-web.js/pull/2692/commits/67bd2dc9ad17954fb285e791ee3cf9383b890994)